portal-kernel/src/com/liferay/document/library/kernel/util/DLValidator.java

```diff
@@ -38,6 +39,8 @@ public interface DLValidator {
 
 	public boolean isValidName(String name);
 
+	public Map<String, Long> getMimeTypeSizeLimit(long groupId);
+
```

Comment:
@thienquach87 
This one is from the auto-generation process. Please make a new commit for this one.

Reply:
will do